tests/components/withings/conftest.py

```diff
@@ -1,15 +1,276 @@
 """Fixtures for tests."""
-
+from collections.abc import Awaitable, Callable, Coroutine
+import time
+from typing import Any
 from unittest.mock import patch
 
+import arrow
 import pytest
+from withings_api.common import (
+    GetSleepSummaryData,
+    GetSleepSummarySerie,
+    MeasureGetMeasGroup,
+    MeasureGetMeasGroupAttrib,
+    MeasureGetMeasGroupCategory,
+    MeasureGetMeasMeasure,
+    MeasureGetMeasResponse,
+    MeasureType,
+    SleepGetSummaryResponse,
+    SleepModel,
+)
 
+from homeassistant.components.application_credentials import (
+    ClientCredential,
+    async_import_client_credential,
+)
+from homeassistant.components.withings.const import DOMAIN
+from homeassistant.config import async_process_ha_core_config
 from homeassistant.core import HomeAssistant
+from homeassistant.setup import async_setup_component
+from homeassistant.util import dt as dt_util
 
-from .common import ComponentFactory
+from . import MockWithings
+from .common import ComponentFactory, new_profile_config
 
+from tests.common import MockConfigEntry
 from tests.test_util.aiohttp import AiohttpClientMocker
 
+ComponentSetup = Callable[[], Awaitable[MockWithings]]
+
+CLIENT_ID = "1234"
+CLIENT_SECRET = "5678"
+SCOPES = [
+    "user.info",
+    "user.metrics",
+    "user.activity",
+    "user.sleepevents",
+]
+TITLE = "henk"
+WEBHOOK_ID = "55a7335ea8dee830eed4ef8f84cda8f6d80b83af0847dc74032e86120bffed5e"
+
+PERSON0 = new_profile_config(
+    "12345",
```

Comment:
Use named attributes (to improve readability) and use a constant for these values (for the same reason).
```
PERSON0_PROFILE = "person0"
PERSON0_USERID = 0

PERSON0 = new_profile_config(
  profile=PERSON0_PROFILE,
  user_id=PERSON0_USERID,
```

While this was moved to this file, it can't hurt to apply good practices.